src/lib/output/renderer.ts

```diff
@@ -199,14 +199,7 @@ export class Renderer extends ChildableComponent<Application, RendererComponent>
                 }
             }
 
-            var filename = Path.join(path, 'theme.js');
```

Comment:
Can you share how that's possible? The code here ran exactly the same regardless of this check.